packages/date/src/DateRange.js

```diff
@@ -132,10 +133,31 @@ const DateRange = ({
     }
   };
 
-  const onFocusChange = async input => {
-    if (!input) {
-      await setFieldTouched(name, true);
+  const syncDates = async () => {
+    if (!metadata.touched) {
+      let value;
+
+      if (!startValue || !endValue) {
+        value = startValue || endValue;
+      }
+
+      if (value) {
+        await setFieldValue(
+          name,
+          {
+            [startKey]: value,
+            [endKey]: value,
+          },
+          true
+        );
+        await setFieldTouched(name, true);
+      }
     }
+  };
+
+  const onFocusChange = async input => {
```

Comment:
When `autoSync` is `true` and we auto fill then end date, we should also close the calendar if the user tabs off of the end date input potentially to prevent it from remaining open until the user has clicked out of the input.

Reply:
I cannot find a way to hook into the `onBlur` event of the end date input.

The `<DateRangePickerInput />` does support an `onEndDateTab` prop. https://github.com/airbnb/react-dates/blob/7c58d6d6aadf0061980f1adeb52465c2606d58ac/src/components/DateRangePickerInput.jsx#L278

However, the `<DateRangePickerInputController />`, which is what the `<DateRangePicker />` renders, does not pass that prop down:

https://github.com/airbnb/react-dates/blob/7c58d6d6aadf0061980f1adeb52465c2606d58ac/src/components/DateRangePickerInputController.jsx#L311-L352

[Here](https://github.com/airbnb/react-dates/blob/master/src/shapes/DateRangePickerShape.js) are all the props that the `<DateRangePicker />` supports. 

I'll open up an issue on `react-dates` unless you see something I'm missing.

Reply:
I think that should help us out here, more so than the `onFocusChange` potentially. Yeah it does look like its missing from the props. We can open up an issue and once that gets fixed we can re-open a new PR to fix this then as its a one off case with the focus on tab change.